### ISSUE TITLE
audioop deprecation

### ISSUE THREAD

**Issue by juanpmarin** [pipecat-ai/pipecat#709]
audioop was removed in python 3.13, making pipecat incompatible with Python >= 3.13

**Comment by rsdouglas** [comment]
Giving this a bump :)

**Comment by markbackman** [comment]
Sorry for being slow on this!
https://github.com/pipecat-ai/pipecat/pull/874

**Comment by markbackman** [comment]
Actually, we're going with: https://github.com/pipecat-ai/pipecat/pull/875

**Comment by markbackman** [comment]
This has been resolved in 0.0.52. Apologies for the delay!

**Comment by luminoso** [comment]
Is this solved @markbackman ? I saw a warning in my logs. 

I've check it up and it is still being imported: https://github.com/pipecat-ai/pipecat/blob/main/src/pipecat/audio/utils.py#L14

What's the expectation? the package manager to block 3.14? or utils.py is not being used anymore?

**Comment by markbackman** [comment]
Yes, this is resolved a long time ago. We use the audioop-lts package for Python >3.12.
